notebooks/407-pedestrian-tracking-webcam/407-pedestrain-tracking.ipynb

```diff
@@ -0,0 +1,617 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    video_file = "data/test.mp4"
</pre><p>please move to notebooks/data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/751/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Got it